src/device/insteonDeviceConfig/configMethods.ts

```diff
@@ -0,0 +1,122 @@
+/*
+	Interfaces & functions used by all the insteon device config nodes
+*/
+
+/* Importing Libraries and types */
+import logger from 'debug';
+/* Configuring logging */
+const debug = logger('node-red-contrib-insteon:configMethods');
+debug.enabled = false;
```

Comment:
This needs to be taken away for production

Reply:
Can do in a later release. This is still a ways away from production ready as it still needs a lot of testing.